moto/acmpca/models.py

```diff
@@ -287,7 +297,7 @@ def not_valid_before(self) -> Optional[float]:
     def import_certificate_authority_certificate(
         self, certificate: bytes, certificate_chain: Optional[bytes]
     ) -> None:
-        self.certificate = x509.load_pem_x509_certificate(certificate)
+        self.certificate_bytes = certificate
```

Comment:
```suggestion
        x509.load_pem_x509_certificate(certificate)
        self.certificate_bytes = certificate
```
Ideally we keep the validation that `certificate`-argument contains a valid X509 certificate. 

As an alternative, something like `assert self.certificate is not None` would also work, as that loads the certificate behind the scenes, but it's less obvious why we're making that assertion.

Reply:
Great catch, added the check and test in ecfa1b336. Thanks @bblommers !